runtime/vm/classallocation.c

```diff
@@ -173,6 +173,18 @@ allocateClassLoader(J9JavaVM *javaVM)
 		/* memset not required as the classLoaderBlocks pool returns zero'd memory */
 
 		classLoader->classHashTable = hashClassTableNew(javaVM, INITIAL_CLASSHASHTABLE_SIZE);
+		
+		/* Create classloader hot field pool and monitor if scavenger dynamicBreadthFirstScanOrdering is enabled */
+		if(javaVM->memoryManagerFunctions->j9gc_hot_reference_field_required(javaVM)) {
```

Comment:
can this be done lazily? Some apps create a lot of classloaders with a small amounts of classes within them

Reply:
???